localbuild/meta.yaml

```diff
@@ -87,6 +87,8 @@ requirements:
     - pycountry
     - websocket-client
     - libtiff <4.5.0
+    - Flask-WTF
```

Comment:
all packages have lower case names, derivated from the example



https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L12